docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,601 @@
+# CA Deployment steps
+
+## Download the binaries
+
+The Fabric CA server and CA client binaries are available for download from [github](https://github.com/hyperledger/fabric-ca/releases). Scroll down to **Assets** and select the latest binary for your machine. The .zip file contains both the CA server and the CA client binaries. Copy the CA server file to a directory where you will run the server. Likewise, copy the CA client binary to its own directory. Having the CA client in its own folder facilitates certificate management especially when you need to interact with multiple CAs.  When you issue a command from the CA client against a CA server, you can target a specific CA by modifying the CA server URL on the request. Therefore, only a single Fabric CA client binary is required and can be used to transact with multiple CAs. More on using the Fabric CA client below.
+
+After you have mastered the use of these binaries, it is likely you will want to use the Fabric CA image for example in a Kubernetes or Docker deployment. For now though, the purpose of this topic is to teach you how to properly use the binaries.
+
+## Fabric CA client
+
+While you can use the Fabric SDKs to interact with your CA, it is recommended that you **use the Fabric CA client to register and enroll node admin identities**. The instructions provided in this topic assume a single Fabric CA client is being used.
+
+**Tip:** While you can run the Fabric CA client binary from anywhere you want, for ease of following these instructions we will refer to it in its own directory named `fabric-ca-client`.
+
+Because you will use the CA client to submit transactions to the CA for enrolling identities, certificate management is critically important when using the CA client. The private and public keys are first generated locally by the Fabric CA client, and then the public key is sent to the CA which returns an encoded "signed certificate". A best practice therefore is for each TLS CA, organization CA, or intermediate CA that you create, to create the following folders:
+- You need a sub-folder in order to connect to each CA, such as `/tls-ca` or `/org1-ca` or `/int-ca`. This folder can be under the Fabric CA client or anywhere that the CA client can access the path. For purposes of these instructions, these folders reside inside the `fabric-ca-client` directory. For example:
+
+  ```
+  mkdir fabric-ca-client
+  cd fabric-ca-client
+  mkdir tls-ca org1-ca int-ca
+  ```
+- Under each CA folder, you need a folder to store the root TLS certificate, such as `org1-ca/tls`. The TLS certificate secures the communications between the CA client and the CA server. This certificate, `ca-cert.pem` is generated after TLS is enabled on the TLS CA and it is subsequently started. Secondly, you need to create a folder to place the generated certificates on an enroll command. If this is an admin identity, it would be  `org1-ca/admin`.
```

Comment:
I tried to address all 3 bullet points here:

Because TLS communications are enabled on a production network, the TLS CA for the organization is responsible for securing communications between all nodes in the organization. Therefore, every time the Fabric CA client transacts with a CA server in that organization, it needs to provide the TLS CA "root certificate" to secure the client-server communications. For example, when the CA client issues a register or enroll request to the CA server, the client request includes that root certificate to perform an SSL handshake.  The TLS CA root certificate, named `ca-cert.pem`, is generated on the TLS CA after TLS is enabled in the server config .yaml file and the server is subsequently started.  To enable TLS communications for your CA client, under each CA folder, you need a sub-folder, such as `org1-ca/tls`, to store the root certificate. Later in this topic, we will copy the root certificate into this folder.  In addition to the `tls` folder, you also need to create a sub-folder for the CA client to store the certificates that are generated by an enroll request. For example, if you are enrolling the admin identity for the `org1-CA`, the folder name would be  `org1-ca/admin`.

Let's confirm this addresses your concerns.

Reply:
makes sense now.